mne/filter.py

```diff
@@ -872,10 +872,11 @@ def filter_data(data, sfreq, l_freq, h_freq, picks=None, filter_length='auto',
 
 
 @verbose
-def create_filter(data, sfreq, l_freq, h_freq, filter_length='auto',
-                  l_trans_bandwidth='auto', h_trans_bandwidth='auto',
-                  method='fir', iir_params=None, phase='zero',
-                  fir_window='hamming', fir_design='firwin', verbose=None):
+def create_filter(data=None, sfreq=None, l_freq=None, h_freq=None,
```

Comment:
I'm actually most interested what you guys think about this change ... that sfreq=None as default ... because I can't have default arguments before non-default arguments

Reply:
I wouldn't set either, seems simpler as having no defaults. Usually you need both (i.e., cater to simplest use case/user) unless you want the advanced mode that returns information provided you know what you are doing, which you can document as the option to pass data=None